src/backend/AuthService/BusinessLogic/Exceptions/CurrentUserDataCheckValidationException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AuthService.BusinessLogic.Exceptions
+{
+    public class CurrentUserDataCheckValidationException : Exception
+    {
+        public CurrentUserDataCheckValidationException(string message)
+         : base(string.Format("Data is invalid at: {0}", message))
```

Comment:
Te `"Data is invalid at"` jest niepotrzebne. Docelowo wpisz po prostu tutaj wiadomość, o której napisałem wcześniej.

Reply:
Changed